css/css-ui-3/cursor-auto-007.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<title>CSS Basic User Interface Test: cursor:auto on unselectable text</title>
+<link rel="author" title="Florian Rivoal" href="https://florian.rivoal.net/">
+<link rel="help" href="http://www.w3.org/TR/css3-ui/#cursor">
+<link rel="help" href="https://drafts.csswg.org/css-ui-4/#content-selection">
```

Comment:
Just a question, this is linking to `css-ui-4` but this folder is `css-ui-3`.
I think the current agreement is that we'll end up having only one folder for each spec,
but letting you know just to be sure this is not a mistake.

Reply:
This is primarily a test about css-ui-3, so it is in the level 3 directory, and has a link to level 3. But it does (optionally, thanks to `@supports`) depend on a level 4 feature, so I'm linking to that too.